### PR TITLE
Intern Task Alina Mulik: Homework

### PR DESCRIPTION
**Pull request by alina-mulik**
Added Hometask4 and reorganized solution

### CODE REVIEW

File: Homework4/Homework4.cs [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+        */
+
+        private static void Task5()
+        {
+            Console.WriteLine("Task 5");
+            string[] namesOfHogwartsStudents = new string[] { "Harry", "Ron", "Neville", "Hello", "Hermione", "Pansy" };
+            foreach (var item in namesOfHogwartsStudents)
+            {
+                if (item == "Hello")
+                {
```
Comment (yauhenists) [inline Homework4/Homework4.cs:114]: Looks like there is a bit misunderstanding of the task:
1) In Console you should see only "Labas!"
2) Stop looping when the condition is met

File: Homework4/Homework4.cs [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+        /* Task 6
+         Создайте массив чисел. Напишите программу, которая подсчитывает сумму первого
+        и последнего элемента массива и выводит ее в консоль.
+         */
+
+        private static void Task6()
+        {
+            Console.WriteLine("Task 6");
+            int[] ints = new int[] { 10, 3, 5, 9, 8, 15, 45 };
+            var summ = ints.First() + ints.Last();
```
Comment (yauhenists) [inline Homework4/Homework4.cs]: Smart)) it works, but could you please rewrite using indexes of the array. Do not use absolute index of the last element (e.g. ints[6]), use Length property

File: Homework4/Homework4.cs [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+
+        /* Task 7
+         Найдите сумму индексов минимального и максимального элементов массива.
+         */
+
+        private static void Task7()
+        {
+            Console.WriteLine("Task 7");
+            int[] ints = new int[] { 1, 0, 5, 9, 8, 99, 15, 7 };
+            var maxIndex = Array.IndexOf(ints, ints.Max());
```
Comment (yauhenists) [inline Homework4/Homework4.cs]: Yep, it works. But please use loops and do not use extension methods (like ints.Max()0

**Review by yauhenists — CHANGES_REQUESTED** [review]

File: Homework4/Homework4.cs (hunk 2 above)
Comment (alina-mulik) [reply Homework4/Homework4.cs]: "what is not prohibited by law is allowed" 😅 Fixed.

File: Homework4/Homework4.cs (hunk 1 above)
Comment (alina-mulik) [reply Homework4/Homework4.cs:114]: Yes, I've misunderstood a bit 🥲  Fixed.

File: Homework4/Homework4.cs (hunk 3 above)
Comment (alina-mulik) [reply Homework4/Homework4.cs]: Understood 😅 Fixed.

File: Homework4/Homework4.cs [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+         */
+
+        private static void Task7()
+        {
+            Console.WriteLine("Task 7");
+            int[] ints = new int[] { 1, 0, 5, 9, 8, 99, 15, 7 };
+            var numOfMax = 0;
+            var numOfMin = 0;
+
+            foreach (var item in ints)
```
Comment (yauhenists) [inline Homework4/Homework4.cs:145]: Yep, it works but a bit complicated. You can create initialize numOfMin and numOfMax with the first element of the array and use 1 for starting with the second element. I;m approving as it is now

**Review by yauhenists — APPROVED** [review]

**Pull request merged**